docs/security/authentication/active-directory/index.md

```diff
@@ -50,9 +53,32 @@ It is possible to use either `NTLM` or `Kerberos` authentication for Active Dire
 
 Without some additional configuration, AD authentication, whether forms-based or integrated, will usually fail on `kerberos` authentication and failback to `NTLM`.
 
+### Supported Setups for Active Directory Authentication {#ActiveDirectoryAuthentication-SupportedAuthentication}
+
+|                                 | Single Octopus Server | High-Availability |
+|---------------------------------|-----------------------|-------------------|
+| NTLM                            |         NTLM          |       NTLM        |
+| Negotiate                       |      Kerberos/NTLM    |       NTLM        |
+| IntegratedWindowsAuthentication |       Kerberos        |       NTLM        |
+
+:::hint
```

Comment:
Nicely put!

> here will be a requirement to run the Octopus Deploy Service as Local System in order to allow for kerberos to succesfully Authenticate.

I could be mistaken but I thought it was a case of requiring Local System or a machine SPN + service account?

Reply:
Yes and no. I would like to get the Repro sorted and tested then revisit this section because the last thing I wanted to do was have some overly confusing instruction. An overly exaggerated example:

_...will either require the use of the Octopus Deploy Service running as a Local System account, in order for Kerberos to Successfully authenticate except in the case of a HA environment where running the OD service as a Local System Accounts setup with server SPNs will still result in duplicate SPNs causing kerberos to fail, so in HA environments it is recommended to use multiple service accounts for the OD service and combining them within a KeyTab file that will be registered as a new SPN tied to the FQDN or NetBios name depending on your setup. Otherwise you can fallback to NTLM._

Please, if you can think of what to put here initially until we confirm each scenario, that'd be great. (hopefully will have this section nailed down by this afternoon.)

Reply:
No probs @danefalvo - having a think

Reply:
Hey @danefalvo I took the liberty of testing that out and making some changes based on what I saw